index.html

```diff
@@ -0,0 +1,109 @@
+                <p>Already have an Intuit account?</p>
+                <a href="#">I forgot my User ID or Password</a> 
+            </div>
+            <span class="main-section">
+                <button type="submit" class="signin-btn"><i class="fa fa-lock" aria-hidden="true">   Sign In</i></button>            </span>
+        </header>
+        <!--End of Header Section--> 
+
+        <!--form section-->
+        <main class="main-section">
+            <form action="#" method="POST">
+                <div class="container-logo">
+                    <img class="img-scl" src="images/intuit-logo.png" alt="logo">
+                </div>
+                <ul class="logos">
+                        <li><a href="#"><img src="images/mint-logo.png" alt="Mint"></a></li>
+                        <li><a href="#"><img src="images/logo-quickbooks.png" alt="Quickbooks"></a></li>
+                        <li><a href="#"><img src="images/logo-turbotax.png" alt="Turbotax"></a></li>
+                </ul>
+                <fieldset class="signup-form">
+                    <fieldset class="form-text">
+                        <p class="form-title">Create an Intuit account</p>
+                        <p class="form-title-2">One account for everything Intuit, including Mint.</p>
+                        <a href="#">Learn more</a>
+                    </fieldset>
+                <fieldset class="space">
+                    <label>Email address</label>
+                    <div>
+                        <input class="input-text" type="email">
+                    </div>
+                </fieldset>
+                <fieldset class="space">
+                    <label>Phone (recommended)</label>
+                    <div>
+                        <input class="input-text" type="tel">
+                    </div>
+                    <p>Standard call, messaging or data rates may apply.</p>
+                </fieldset>
+                <fieldset class="space">
+                    <label>Password</label>
+                    <div>
+                        <input class="input-text" type="password">
+                    </div>
+                </fieldset>
+                <span>
+                    <button type="submit" class="submit-btn"><i class="fa fa-lock" aria-hidden="true"> Create Account</i></button>
+                </span>
+                    <div class="form-bottom-text">
+                        <p>
+                            <i>By clicking Create account, you agree to our <br>
+    <a href="#">Terms</a> and have read and acknowledge our <a href="#">Privacy <br> Statement</a>.</i>
+                        </p>
+                    </div>
+                </fieldset>                 
+            </form>
+        </main>
+        <div class="captcha">
+            <fieldset class="form-footer">
+                <p>Invisible reCAPTCHA by Google <a href="#">Privacy Policy</a> and <a href="#">Terms of Use.</a> </p>
+            </fieldset>
```

Comment:
- [ ] **[MANDATORY]** Please check this problem, this element is not properly sized:
![image](https://user-images.githubusercontent.com/53256139/87486794-b0b2db80-c601-11ea-848a-843a20ac3f71.png)